storefront/tokens/tokens.css

```diff
@@ -0,0 +1,181 @@
+/**
```

Comment:
Maybe the just use the actual token files from [@digdir/design-system-tokens](https://www.npmjs.com/package/@digdir/design-system-tokens)? Would be easier to keep updated as this file is already outdated.

We could also generate a separate json format specifically for showcasing the tokens in the storefront, but that would be a separate issue.

Reply:
Added a separate issue for this: #343